### DISCUSSION TITLE
Bypass Zones

### DISCUSSION THREAD

**Discussion by BeardOnTheRange** [ParadoxAlarmInterface/pai#400] [Q&A] [answered]
Hi All, 

This addon worked great for me and was relatively easy to setup. However, I have now hit an issue with some sensors on my alarm, which for the time being necessitates that I bypass some zones before arming to avoid false alarms. 

I cannot figure out how to do this via the addon, I have been looking through the wiki but can't seem to find a quick way that one can by pass a zone when editing. Can anyone assist?

**Comment by yozik04** [comment, marked as answer]
If you arm from keypad then you can't. You can not bypass a zone while arming and when armed. If you arm from HA them you can write a script so it would bypass first then do arming. If you want to bypass entities be available in HA then make sure you add `bypassed` to `HOMEASSISTANT_PUBLISH_ZONE_PROPERTIES` array. https://github.com/ParadoxAlarmInterface/pai/wiki/Home-Assistant-integration

**Reply by BeardOnTheRange** [reply]
Thanks, not sure how I missed this. But it solved my problem.

**Comment by elvtechnology** [comment]
If you have installer access, you can disable the sensor which will avoid the unneccessary hassle of having to bypass each time,
